planner_cspace/src/planner_3d.cpp

```diff
@@ -859,8 +872,12 @@ class Planner3dNode
     }
     ROS_DEBUG("Cost estimation cache search queue initial size: %lu, capacity: %lu",
               dm_debug.search_queue_size, dm_debug.search_queue_cap);
-    ROS_DEBUG("Cost estimation cache updated (%0.4f sec.)",
-              boost::chrono::duration<float>(tnow - ts).count());
+    const float dur = boost::chrono::duration<float>(tnow - ts).count();
+    ROS_DEBUG("Cost estimation cache updated (%0.4f sec.)", dur);
+    metrics_.data.push_back(neonavigation_metrics_msgs::metric(
+        "distance_map_dur",
```

Comment:
[optional] Both the generation and the update of the cache are reported in `distance_map_dur`. It might be helpful to have separate metric if we want to distinguish them.